### PR TITLE
Add removing of unnecessary files for CentOS builds

### PR DESCRIPTION
**Pull request by louisrokitta**
This change removes unnecessary files if yum install is used

### CODE REVIEW

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -234,6 +234,13 @@ rm %{buildroot}%{_sbindir}/katello-tracer-upload
 rm %{buildroot}%{plugins_confdir}/tracer_upload.conf
 %endif
 
+%if %{yum_install}
```
Comment (ekohl) [inline packages/client/katello-host-tools/katello-host-tools.spec:237]: Shouldn't this be something like:
```suggestion
%if !%{zypper_install}
```

Or is that not supported?

Comment (louisrokitta) [reply packages/client/katello-host-tools/katello-host-tools.spec]: Thanks for your suggestion. I tested it and it works.

**Comment by ekohl** [comment]
ok to test

**Comment by louisrokitta** [comment]
Rebased to rpm/develop

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 2]
```diff
@@ -234,6 +234,13 @@ rm %{buildroot}%{_sbindir}/katello-tracer-upload
 rm %{buildroot}%{plugins_confdir}/tracer_upload.conf
 %endif
 
+%if !%{zypper_install}
+rm %{buildroot}%{katello_libdir}/zypper_tracer.py*
```
Comment (ekohl) [inline packages/client/katello-host-tools/katello-host-tools.spec:238]: Looks like this doesn't work?
```
+ rm '/builddir/build/BUILDROOT/katello-host-tools-3.5.5-3.el7.noarch/usr/lib/python2.7/site-packages/katello/zypper_tracer.py*'
rm: cannot remove '/builddir/build/BUILDROOT/katello-host-tools-3.5.5-3.el7.noarch/usr/lib/python2.7/site-packages/katello/zypper_tracer.py*': No such file or directory
```

**Comment by pcreech** [comment]
Closing due to inactivity.  If you feel it needs to be reopened, please reopen it!

**Pull request closed without merging**